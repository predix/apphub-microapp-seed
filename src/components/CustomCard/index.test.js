import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Card from '.';

describe('Ajax Card', () => {
  test('should render', () => {
    const wrapper = shallow(<Card headerText="My Header" />);
    expect(wrapper.find('.px-card')).toHaveLength(1);
  });
  test('should match snapshot', () => {
    const tree = renderer.create(<Card headerText="My Header">Content</Card>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
