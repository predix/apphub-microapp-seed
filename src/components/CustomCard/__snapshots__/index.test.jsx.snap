// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ajax Card should match snapshot 1`] = `
<div
  className="px-card custom-card"
>
  <header
    onClick={[Function]}
    role="presentation"
  >
    <span
      className="epsilon caps"
    >
      My Header
    </span>
  </header>
  <section>
    Content
  </section>
</div>
`;
